src/templates/settings/export.html

```diff
@@ -61,5 +61,12 @@
         <div class="gif-upload-status"></div>
       </div>
     </div>
+    <div class="settings-title">
+      Export as C File
+    </div>
+    <div class="settings-item">
+      <div class="settings-description">C file with frame rendered as array.</div>
+      <button type="button" class="button button-primary c-download-button">Download C file</button>
+    </div>
```

Comment:
I think we should temporarily move this section above the "GIF export" section. When using the "Get public URL" button of the GIF export, a preview of the image is inlined and makes the new section move up and down :)